lms/djangoapps/ccx/views.py

```diff
@@ -367,17 +398,22 @@ def ccx_invite(request, course):
                 unenroll_email(ccx, email, email_students=email_students)
         except ValidationError:
             log.info('Invalid user name or email when trying to invite students: %s', email)
-    url = reverse('ccx_coach_dashboard', kwargs={'course_id': course.id})
+    url = reverse(
+        'ccx_coach_dashboard',
+        kwargs={'course_id': CCXLocator.from_course_locator(course.id, ccx.id)}
```

Comment:
I think this is a six-of-one kind of question.  Yes, we need to construct a CCXLocator from values that were previously derived from a CCXLocator.  But in most of these views we also need to have the course descriptor, so is it cheaper, more DRY to do that operation once and this one many times, or that one many times and this once?  I think its a toss-up.